app/views/users/show.html.erb

```diff
@@ -1 +1,31 @@
 <h1>User Information</h1>
+<div class="user-card flex">
+  <div class="user-image border">
+    <img src="<%= @user.photo %>" alt="<%= @user.name %>'s photo" width="100% heigth="100%">
+  </div>
+  <div class="user-info border">
+    <h2><%= @user.name %></h2>
+    <p class="posts-number">Number of posts: <%= @user.posts.count %></p>
+  </div>
+</div>
+<div class="user-card left">
+  <h3>Bio:</h3>
+  <br>
+  <p><%= @user.bio %></p>
+</div>
+<div>
+   <ul>
+    <% @user.posts.limit(3).each do |post| %>
```

Comment:
- [ ] Great job so far @ClaudiaRojasSoto There are a few changes I will suggest you implement and you'll be on your way to the next Milestone :wink: 
- [ ] Great job limiting the number of posts to 3. However, I will suggest that you make use of `three_most_recent_posts` in your  `user.rb` model here instead since that was the purpose of the method to be used in your project :100:

Reply:
Done. Thank you @NahnahAJ